app/src/main/java/com/imashnake/animite/features/home/HomeScreen.kt

```diff
@@ -87,88 +87,76 @@ fun HomeScreen(
     when {
         rows.all { it.first is Resource.Success } -> {
             val scrollState = rememberScrollState()
-            TranslucentStatusBarLayout(
-                scrollState = scrollState,
-                distanceUntilAnimated = dimensionResource(R.dimen.banner_height)
-            ) {
+            TranslucentStatusBarLayout(scrollState) {
                 Box(
                     modifier = Modifier
                         .verticalScroll(scrollState)
                         .navigationBarsPadding()
                 ) {
-                    Box {
-                        Image(
-                            painter = painterResource(R.drawable.background),
-                            contentDescription = null,
-                            modifier = Modifier
-                                .fillMaxWidth()
-                                .height(dimensionResource(R.dimen.banner_height))
-                                .bannerParallax(scrollState),
-                            contentScale = ContentScale.Crop,
-                            alignment = Alignment.TopCenter
-                        )
+                    BannerLayout(
+                        banner = { bannerModifier ->
```

Comment:
Can we avoid this "please use the modifier" pattern somehow? We've got a few cases at work and it's not fun

Reply:
Documented :)